dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/Event.java

```diff
@@ -124,6 +124,10 @@
 
     private String assignedUserDisplayName;
 
+    private String assignedUserFirstName;
```

Comment:
Maybe I got it wrong, but I thought we agreed to use the userinfo object for assignedUser as well? And extend it with uid? that way we dont end up with a long list of properties like this.

Reply:
I think it is a User(Info) property in the tracker domain. Should this here in (dxf2) be a UserInfoSnapshot like `createdByUserInfo` ?

Reply:
@stian-sandvold we are. This is the old importer and I needed to add these 2 fields as we were only getting them together from the DB.
We are using this DTO to then mapping it to our domain model for new tracker export and in our domain we are using the User info object as agreed

Reply:
Ok, nevermind me then 🙈 Thanks for clearing that up :)